### DISCUSSION TITLE
Error: User not allowed to create organizations

### DISCUSSION THREAD

**Discussion by liferadioat** [dani-garcia/vaultwarden#3556] [Q&A]
Hi,

I´m owner of several organizations and admin at a docker hosted vaultwarden, but I can´t create new organizations. Error: user not allowed to create organizations. Can anyone tell me please, how to solve it? 

Thanks and regards,

**Comment by BlackDex** [comment]
You probably have some settings configured which prevents this.

See: https://github.com/dani-garcia/vaultwarden/blob/445680fb84d238aa432fdec271235933043410e4/.env.template#L254-L260

**Comment by liferadioat** [comment]
Thanks, yes you´re right. When using docker it seems not to work:

`docker run -d \
--name='vaultwarden' \
--restart=always \
        -e ADMIN_TOKEN=xxx \
        -e SIGNUPS_ALLOWED=false \
        -e SIGNUPS_DOMAINS_WHITELIST=xxx.at \
        -e ORG_CREATION_USERS=test@xxx.at,test2@xxx.at \
`
Is this .env parameter supported at docker?

**Reply by BlackDex** [reply]
All env parameters are supported as docker env, or just global envs, as long as they are env's 😉

**Reply by liferadioat** [reply]
OK, so my example should work or is there any mistake?

**Reply by BlackDex** [reply]
What do the logs say?

**Comment by Stitch10925** [comment]
I have the ORG_CREATION_USERS variable set to the user I log in with, however, when creating an organization an error occurs:

`[WARN] Data guard `JsonUpcase < OrgData >` failed:`
` Error("missing field `ExternalId`", line: 1, column: 2717)`
`[response][INFO] (create_organization) POST /api/organizations => 422 Unprocessable Entity`

This is for version 2023.5.0:testing running in Docker

**Reply by BlackDex** [reply]
Thanks for reporting, while this has nothing to do with the OP, i did fixed it quickly

**Reply by Stitch10925** [reply]
Thanks! I will pull the latest version.
